<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_pos_graph_config</class>
 <widget class="QDialog" name="Dialog_pos_graph_config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>404</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>442</width>
    <height>404</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>421</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Ancla:</string>
   </property>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>43</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Activar modo pasivo:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>41</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Activar</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>421</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Ejes</string>
   </property>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rango min Eje X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rango min Eje Y:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_9">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-30.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>62</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rango max Eje Y:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_10">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-30.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_11">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-30.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_12">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-30.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>32</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rango max Eje X:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>421</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Anclas:</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ancla 1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ancla 2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ancla 3:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ancla 4:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>26</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-30.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>58</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-30.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>88</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-30.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>118</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-30.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>26</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-30.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>56</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-30.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_7">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>86</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-30.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_8">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>116</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-30.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>29</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>59</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>89</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>119</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>28</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>58</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>89</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>119</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_pos_graph_config</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_pos_graph_config</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
